azure-pipelines.yml

```diff
@@ -39,6 +39,8 @@ variables:
     value: "Tiberius"
```

Comment:
One last comment I forgot to put earlier. There are different versions of packages being used in different projects in the repo. It would be worth consolidating these so that all projects use the same packages.

Reply:
This is still to do.